src/libMessage/ZilliqaMessage.proto

```diff
@@ -1048,6 +1048,11 @@ message ConsensusCollectiveSig
     ConsensusInfo consensusinfo = 1;
     ByteArray pubkey            = 2;
     ByteArray signature         = 3;
+    oneof oneof1
```

Comment:
I think `oneof` isn't necessary. There are 2 purposes for `oneof` in ZilliqaMessage: (1) to preserve proto2 data format by force-adding a value even if it's default (e.g., for primitive numeric types), and (2) in case there are multiple possible data types (like `union` in C).

Reply:
I thought its also used for specifying optional field. The `FinalCollectiveSig` is not going to have `newannouncement` field.